client/app/config/index.js

```diff
@@ -16,7 +16,6 @@ import ngMessages from 'angular-messages';
 import toastr from 'angular-toastr';
 import ngUpload from 'angular-base64-upload';
 import vsRepeat from 'angular-vs-repeat';
-import 'angular-moment';
```

Comment:
👋

Reply:
I think it's the first Angular dependency we remove